File: DocumentProject.WebClient/src/app/_models/organization.ts
import { BaseModel } from "./abstract/base-model";
import { Application } from "./application";
import { Manager } from "./manager";
import { Member } from "./member";

export class Organization extends BaseModel {
  name: string | undefined;
  type: string | undefined;
  bin: string | undefined;
  address: string | undefined;
  contactNumber: string | undefined;

  ownerManagerId: string | undefined;
  ownerManager: Manager | undefined;

  applications: Application[] = []
  members: Member[] = []
}

File: DocumentProject.WebAPI/DocumentProject.WebAPI/Data/Organization.cs
using DocumentProject.WebAPI.Data.Abstract;

namespace DocumentProject.WebAPI.Data
{
    public class Organization : Entity
    {
        public string Name { get; set; }
        public string Type { get; set; }
        public string BIN { get; set; }
        public string Address { get; set; }
        public string ContactNumber {  get; set; }
        public Guid? OwnerManagerId { get; set; }
        public Manager? OwnerManager { get; set; }

        public List<Member> Members { get; set; }
        public List<Application> Applications { get; set; }
    }
}
